docs/states.md

```diff
@@ -0,0 +1,245 @@
+| `before:`            | `&::before`                                                    |
+| `after:`             | `&::after`                                                     |
+| `first-letter:`      | `&::first-letter`                                              |
+| `first-line:`        | `&::first-line`                                                |
+| `marker:`            | `&::marker`                                                    |
+| `selection:`         | `&::selection`                                                 |
+| `file:`              | `&::file-selector-button`                                      |
+| `backdrop:`          | `&::backdrop`                                                  |
+| `placeholder:`       | `&::placeholder`                                               |
+| `sm:`                | `@media (min-width: 480px)`                                    |
+| `md:`                | `@media (min-width: 768px)`                                    |
+| `lg:`                | `@media (min-width: 990px)`                                    |
+| `xl:`                | `@media (min-width: 1330px)`                                   |
+| `dark:`              | (unsupported) `@media (prefers-color-scheme: dark)`            |
+| `portrait:`          | (unsupported) `@media (orientation: portrait)`                 |
+| `landscape:`         | (unsupported) `@media (orientation: landscape)`                |
+| `motion-safe:`       | (unsupported) `@media (prefers-reduced-motion: no-preference)` |
+| `motion-reduce:`     | (unsupported) `@media (prefers-reduced-motion: reduce)`        |
+| `contrast-more:`     | (unsupported) `@media (prefers-contrast: more)`                |
+| `contrast-less:`     | (unsupported) `@media (prefers-contrast: less)`                |
+| `print:`             | (unsupported) `@media print`                                   |
+| `supports-[…]:`      | (unsupported) `@supports (…)`                                  |
+| `aria-checked:`      | (unsupported) `&[aria-checked=“true”]`                         |
+| `aria-disabled:`     | (unsupported) `&[aria-disabled=“true”]`                        |
+| `aria-expanded:`     | (unsupported) `&[aria-expanded=“true”]`                        |
+| `aria-hidden:`       | (unsupported) `&[aria-hidden=“true”]`                          |
+| `aria-pressed:`      | (unsupported) `&[aria-pressed=“true”]`                         |
+| `aria-readonly:`     | (unsupported) `&[aria-readonly=“true”]`                        |
+| `aria-required:`     | (unsupported) `&[aria-required=“true”]`                        |
+| `aria-selected:`     | (unsupported) `&[aria-selected=“true”]`                        |
+| `aria-[…]:`          | (unsupported) `&[aria-…]`                                      |
+| `data-[…]:`          | (unsupported) `&[data-…]`                                      |
```

Comment:
Just curious, do you know if we should support these modifiers in the future?

Reply:
We should prob support some of these in the future, yes